docs/alternatives/navigator_extension_alternative.bs

```diff
@@ -45,6 +45,41 @@ Markup Shorthands: markdown yes, css no
 
 </div>
 
+# Key terms # {#key-terms}
+
+The <dfn for="web environment">web environment</dfn> is defined as <i>TODO</i>
+
+# Attesters # {#attester}
+
+The term <dfn for="attester">attester</dfn> refers to a third party capable of returning an Web
+Environment Integrity verdict. A <dfn for="verdict">Web Environment Integrity verdict</dfn> refers
+to an response that confirms if the [=attester=] trusts the [=web environment=] the [=user agent=] is
+executing in.
+
+The [=user agent=] connects to the [=attester=] through an {{AttesterConnection}}.
+
+The [=user agent=] SHOULD use separate {{AttesterConnection}}s if the {{AttesterConnection}}
+stores state in the [=verdict=] that can be used for cross site tracking.
+
+## Token Format ## {#attester-tokens}
+
+<i>Todo</i>
+
+## Browser Acceptance Requirements ## {#attester-browser-requirements}
+
+<i>Todo</i>
+
+## {{AttesterConnection}} ## {#attester-connection}
+
+<xmp class="idl">
+  interface AttesterConnection {
+    ArrayBuffer getAttestation(DOMString contentBinding);
+  };
+</xmp>
+
+: getAttestation
+:: Returns a COSE signed CBOR object as an ArrayBuffer from the [=attester=] that contains the [=verdict=].
```

Comment:
is this verdict the `Environment Integrity verdict`?

Reply:
Yeah, updated to make the reference the exact same field for clarity. Also updated the name to Integrity verdict to make it less of a mouthful.